portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -22,44 +22,45 @@
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content. TODO: modify this file 
+to handle user input instead of hard-coded comments*/
 @WebServlet("/data")
 public final class DataServlet extends HttpServlet {
 
-	@Override
-    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        
-        PortfolioComments commentsRecord = new PortfolioComments();
+  /** Takes in hard-coded messages and store them in json **/
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  
+    PortfolioComments commentsRecord = new PortfolioComments();
+    StringBuilder json = new StringBuilder();
 
-        // Add Sample Comments
-        commentsRecord.addComment("You rock! Keep on coding!");
-        commentsRecord.addComment("This is momma! You got this girl :-)");
-        commentsRecord.addComment("This is dad! Go break some eggs, babygirl.");
+    // Add Sample Comments
+    commentsRecord.addComment("You rock! Keep on coding!");
+    commentsRecord.addComment("This is momma! You got this girl :-)");
+    commentsRecord.addComment("This is dad! Go break some eggs, babygirl.");
 
-	    // Create commentsRecord object in json form
-        String json = convertToJson(commentsRecord);
+	// Create commentsRecord object in json form
+    convertToJson(commentsRecord);
 
-        // Send the JSON as the response
-        response.setContentType("application/json;");
-        response.getWriter().println(json);
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json.toString());
+  }
 
-    }
-
-    /**
-    * Converts the commentsRecord instance into a JSON string using manual String concatentation.
-    */
-    private String convertToJson(PortfolioComments commentsRecord) {
-        String json = "{";
-        json += "\"Comment0\": ";
-    	json += "\"" + commentsRecord.getComment(0) + "\"";
+  /**
+  * Converts the commentsRecord instance into a JSON string using manual String concatentation.
+  */
+  private void convertToJson(PortfolioComments commentsRecord) {
+    json.append("{")
```

Comment:
there is no json field declared for this method.

Reply:
I thought you wanted me to make it a global variable? Right now, its a global variable and this method just adds onto it.

Reply:
Converted to gson, nevermind! :)